### DISCUSSION TITLE
Enhancement: Compile the game without Visual Studio.

### DISCUSSION THREAD

**Discussion by MrGenie151** [KadeArchive/Kade-Engine#571] [Q&A]
#### Please check for duplicates or similar issues before creating this issue.
I don't think there are any... But also i read that u needed to do this for windows on the github page but i cant be sure
## What is your suggestion, and why should it be implemented?
I am going to use Atom as my editor of choice to edit the .hx files and according to the github page i have to install Visual Studio Code 2019 in order to build to windows. however, i **REALLY** do not want to do this as something i will not even use will take up space on my computer. Is there a way around this that you could tell me or can you patch it out in the next update?

**Comment by isakube** [comment]
no. vsc is pretty much the only way, and if it isn't then it's the simplest.

**Comment by puyoxyz** [comment]
`lime build windows -neko` MIGHT work. I'm **not sure** (and can't test as I'm not on windows). If it doesn't, there's no way around it other than switching to macOS or Linux.

Also, it doesn't say to install VSCode. It says to install VS 2019. They're different.

(if you try `lime build windows -neko` without visual studio installed please tell me if it works or it doesn't thanks)
